packages/itwinui-css/src/utils/notification-marker/notification-marker.scss

```diff
@@ -14,8 +14,8 @@ $iui-notification-marker-border-width: var(--iui-size-3xs);
     top: calc(var(--iui-size-s) * -0.5);
     right: calc(var(--iui-size-s) * -0.5);
     border-radius: 100%;
-    background-color: var(--iui-color-foreground-#{$status});
-    border: $iui-notification-marker-border-width solid var(--iui-color-background-1);
+    background-color: var(--iui-color-icon-#{$status}); // TODO this won't work with white
```

Comment:
@mayank99 this won't work with `--iui-color-white`.

Reply:
what does this have to do with `iui-color-white`?

Reply:
Notification marker's `classes.scss` originally supported `accessory` which is now replaced with `white`:
```scss
@each $status in positive, warning, negative, accessory {
  &[data-iui-variant='#{$status}'] {
    @include iui-notification-marker-status($status);
  }
}
```

The color of the notification dot is set here:
```scss
--_iui-notification-marker-color: var(--iui-color-icon-#{$status});
```

But we don't have a `--iui-color-icon-white`, only `--iui-color-white`.  I think I can come up with a fix using `@if`.